libs/utils/utils.js

```diff
@@ -685,6 +685,7 @@ export async function loadIms() {
       onError: reject,
     };
     loadScript('https://auth.services.adobe.com/imslib/imslib.min.js');
+    console.log('ims load start', performance.now() - window.utilsloaded);
```

Comment:
not sure if appropriate, but did we want to add lana logging here?

Reply:
doh - left in some perf timing I was testing. Thanks